src/lib/protocols/hangout.c

```diff
@@ -84,12 +84,13 @@ static u_int8_t is_google_flow(struct ndpi_detection_module_struct *ndpi_struct,
 void ndpi_search_hangout(struct ndpi_detection_module_struct *ndpi_struct,
 			 struct ndpi_flow_struct *flow) {
   struct ndpi_packet_struct * packet = &flow->packet;
+  int matched_src = 0;
```

Comment:
Scope of `matched_src` can be reduced if I am not wrong. (reading source code from a smartphone is sometimes confusing)

Reply:
Done in the last push. Thanks